src/AutoRest.CSharp/Common/Output/Models/Requests/RestClientMethod.cs

```diff
@@ -13,7 +13,7 @@ namespace AutoRest.CSharp.Output.Models.Requests
 {
     internal class RestClientMethod
     {
-        public RestClientMethod(string name, string? summary, string? description, CSharpType? returnType, Request request, IReadOnlyList<Parameter> parameters, Response[] responses, DataPlaneResponseHeaderGroupType? headerModel, bool bufferResponse, string accessibility, InputOperation operation)
+        public RestClientMethod(string name, string? summary, string? description, CSharpType? returnType, Request request, IReadOnlyList<Parameter> parameters, Response[] responses, DataPlaneResponseHeaderGroupType? headerModel, bool bufferResponse, string accessibility, InputOperation operation, IReadOnlyList<Parameter>? originalParameters = null)
```

Comment:
Can we use one input for parameters and internally decide if we use the property bag?  We would decide if we want the bag, if so we would make a copy of parameters and modify it and expose it via the `Parameters` property, and expose the original as `OriginalParameters`.

This would make it easier to use across generator types.

We could then have a flag which defaults to true for mpg and dpg, but false for hlc.  Any single library could override that default.

If its easier for first release to have dpg default to false as well that could work

Reply:
fixed, and for now I just enabled property bag for mgmt plane.